File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    
    // gets user input
    do
    {
        n = get_int("Height: ");
    } 
    while (n < 0 || n > 23);
    
    for (int i = 0; i < n; i++)
    {
        // prints spaces
        for (int j = i + 1 ; j < n; j++)
        {
            printf(" ");
        }
        // prints #s
        for (int k = 0; k < i + 2; k++)
        {
            printf("#");
        }

        // prints new line
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int


def main():
    while True:
        h = get_int("Height: ")
        w = h
        if h > 0 and h < 9:
            break

    for i in range(1, h + 1):
        nh = i
        ns = w - nh
        # print spaces without break line
        print(" " * ns, end="")
        # print hashes
        print("#" * nh)


if __name__ == "__main__":
    main()